wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -29,31 +29,29 @@ class WagnGenerator < Rails::Generators::AppBase
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
   def dev_setup
     # TODO: rename or split, gem_path points to the source repo, card and wagn gems are subdirs
-    @gem_path = options['gem-path'] || "ENV['WAGN_DEV_GEM_PATH']"
+    @gemfile_gem_path = @gem_path = options['gem-path']
+    env_gem_path = ENV['WAGN_DEV_GEM_PATH']
+    if env_gem_path.present?
+      @gemfile_gem_path = %q{#{ENV['WAGN_DEV_GEM_PATH']}}
+      @gem_path = env_gem_path
+    end
+    
     @include_jasmine_engine = false
     if options['core-dev']
+      unless @gem_path
+        @gemfile_gem_path = @gem_path = ask("Enter the path to your local wagn gem installation: ")
+      end
+
       @include_jasmine_engine = true
       @spec_path = @gem_path
       @spec_helper_path = File.join @spec_path, 'card', 'spec', 'spec_helper'
-      # Will this work when @gem_path is 'ENV[...]' ?
-      @features_path = File.join @gem_path, 'wagn/features/'  # ending slash is important in order to load support and step folders
-      @simplecov_config = "card_core_dev_simplecov_filters"
-      template "rspec", ".rspec"
-    elsif options['mod-dev']
```

Comment:
this was weird, there were two `elsif options['mod-dev']` sections.  I haven't really tested these cases, but I tried to get rid of the cruft.